### ISSUE TITLE
Training accuracy remains zero

### ISSUE THREAD

**Issue by rainmaker29** [DequanZhu/FaceNet-and-FaceLoss-collections-tensorflow2.0#6]
Accuracy remains zero on training. The loss also remains constant .

**Comment by LawsonLv** [comment]
Training acc over epoch 1: 0.0000;
Me too, always zero
